tests/phpunit/src/Commands/Pull/PullCommandTestBase.php

```diff
@@ -15,6 +16,8 @@
  */
 abstract class PullCommandTestBase extends CommandTestBase {
 
+  use IdeRequiredTestTrait;
```

Comment:
Maybe this isn't best. If it's only used for a few test cases, it should be a helper not a trait.